contracts/Dice.vy

```diff
@@ -0,0 +1,74 @@
+from vyper.interfaces import ERC20
+
+# Define constants and types
+MIN_SLIDER: constant(int128) = 1
+MAX_SLIDER: constant(int128) = 100
+
+interface RandomNumberInterface:
+    def get_random_number() -> int128: view
```

Comment:
We should probably call out in a README or whomever is writing this that they need to link this code.